src/features/venue/pages/VenueBody.tsx

```diff
@@ -103,7 +104,10 @@ export const VenueBody: FunctionComponent<Props> = ({ venueId, onScroll }) => {
 
         {/* Description */}
         <HiddenTitle>{t`Description`}</HiddenTitle>
-        <PartialAccordionDescription description={description || ''} />
+        <VenuePartialAccordionDescription
+          description={description || ''}
```

Comment:
le type de la `props` étant ` description?: string | undefined`

je propose
```suggestion
          description={description || undefined}
```

voir meme si TypeScript est ok
```suggestion
          description={description}
```